MekHQ/src/mekhq/campaign/parts/MissingBattleArmorSuit.java

```diff
@@ -253,14 +251,13 @@ public void fix() {
                 }
                 missingStuff.add(missingBaEquip);
             }
-            for(int childId : replacement.getChildPartIds()) {
-        		Part childPart = campaign.getPart(childId);
-        		if(childPart instanceof BaArmor && null != origArmor) {
+            for (Part childPart : replacement.getChildParts()) {
+        		if (childPart instanceof BaArmor && null != origArmor) {
         			unit.getEntity().setArmor(((BaArmor)childPart).getAmount(), trooper);
         			origArmor.updateConditionFromEntity(false);
```

Comment:
This looks odd, likely tab vs space issues

Reply:
Agreed. Because of the `//region` code used, I cannot use formatting anymore with VSCode or I'll break them. I'll see what I can do to switch the file to spaces without breaking region comments.